woof-code/rootfs-packages/ptheme/usr/sbin/ptheme_gtk

```diff
@@ -180,7 +180,12 @@ while [ $# != 0 ]; do
 		case $1 in
 			-t) export THEME="$2"
 				[ ! "$THEME" ] && export THEME="$ACTIVE_THEME"
-				shift;;
+				shift
+				EXIT_WITHOUT_GUI=1;;
```

Comment:
I think you should move this to line 176, so `EXIT_WITHOUT_GUI` is defined even if neither -t nor -T is specified

Reply:
It would never be needed then, since `THEME` would not be set, I suppose. Still correcting this, it does not hurt to do so 👍.